.github/actions/setup-env-vars/action.yml

```diff
@@ -0,0 +1,34 @@
+name: "Setup Env Vars"
```

Comment:
Is it a duplicate from the AutoGluon? If so, can we reuse the same so we could avoid duplicate codes and changes in 2 places going forward? Something like [this](https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow)

Reply:
Yes, it's duplicated. Not sure if we can use it directly, but I can give it a shot.

Reply:
OK. I just researched it a little bit. What you were referring is `reusable workflow`, while this is a `composite action`. `composite action` cannot be used across different repos.

We might consider replacing this to be a reusable workflow later (if possible), but I'd like to not do it in this PR

Reply:
did some research, I think the best way is to create starter [workflow](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization) for common steps for reuse. Fine to proceed with current approach